Documentation/cmdref/cilium_identity_list.md

```diff
@@ -15,6 +15,7 @@ cilium identity list [LABELS] [flags]
 ### Options
 
 ```
+      --endpoints       list identities of locally managed endpoints
```

Comment:
I was a little surprised that this listed the refcount and not the actual endpoints using each identity. I don't know what to name this instead, though, apart from `used-locally` or something equally obtuse. Do we ever anticipate adding a column with the endpoint IDs that have each identity?

Reply:
I'm open to changing the flag. `--used-locally` sounds fine to me. I can change that in a follow-up.

Reply:
I had the same initial thought. That would be a nice extension to this.